gendoc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I've `set -u` since `wget` returns `8` ( https://www.gnu.org/software/wget/manual/html_node/Exit-Status.html ) which is fine, but I can't then check `$?` because the script terminates before I can. In addition, I'm relying on `curls` exit code being non-zero whilst testing if the server is up.